### PR TITLE
Update UG & DG of edit session, edit schedule and UG symbols

### PR DESCRIPTION
**Pull request by maguireong**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -213,6 +213,75 @@ The following activity diagram summarizes what happens when a user executes a ne
...
+##### Aspect: How edit schedule executes
+
+* **Alternative 1 (current choice):** Retrieve Schedule using Client and Session Index.
+  * Pros: More troublesome to implement. Clearer to retrieve.
+  * Cons: Require user to know the Client and Session Index separately.
+
+* **Alternative 2:** Retrieve Schedule using Schedule Index
+  itself.
+  * Pros: Easier to retrieve.
+  * Cons: Implementation is more confusing as User there's a conflict between Index and user-typed String index.
```
Comment (kelvinvin) [inline docs/DeveloperGuide.md:414]: confusing to User

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -427,6 +496,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2a1. FitEgo shows an error message.
 
       Use case resumes at step 2.
+    
+* 2b. User edits session information (i.e. gym, exercise type, start time and duration).
```
Comment (kelvinvin) [inline docs/DeveloperGuide.md]: Is this the same as MSS step 3?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -565,25 +642,25 @@ testers are expected to do more *exploratory* testing.
...
       Details of the edited schedule is shown in the status message. Timestamp in the status bar is updated.
       
-   1. Test case: `editschedule c/1 s/1 pd/true`<br>
+   1. Test case: `schedit c/1 s/1 pd/true`<br>
          Expected: Edit Schedule with client index 2 and session index 1 is edited to be paid. 
          In the right panel, the client's name in the related session will be indicated as green. 
          Details of the edited schedule is shown in the status message. Timestamp in the status bar is updated.
          
-   1. Test case: `editschedule c/1 s/1 pd/false`<br>
+   1. Test case: `schedit c/1 s/1 pd/false`<br>
```
Comment (dhafinrazaq) [inline docs/DeveloperGuide.md]: now pd/paid or pd/unpaid

File: docs/UserGuide.md [hunk 4]
```diff
@@ -214,6 +223,13 @@ Examples:
 * `cadd n/Jane Doe p/91234567 e/jane@gmail.com`
 * `cadd n/John Doe p/91231367 e/jojo@gmail.com t/injured-thigh`
 
+
+### Listing all clients : `clist`
+ƒhow
+Shows a list of all clients in the clients list.
+
+Format: `clist`
+
```
Comment (tanweijie123) [inline docs/UserGuide.md]: clist exist in 201-208

**Review by tanweijie123 — APPROVED** [review]
👍

**Pull request merged**